lib/improver/psychrometric_calculations/psychrometric_calculations.py

```diff
@@ -72,15 +72,14 @@ def specific_heat_of_moist_air(mixing_ratio):
         dry air and water vapour in proportion given by the specific humidity.
 
         Args:
-            mixing_ratio (iris.cube.Cube):
+            mixing_ratio (numpy.ndarray):
                 Cube of specific humidity (fractional).
         Returns:
-            iris.cube.Cube:
+            numpy.ndarray:
                 Specific heat capacity of moist air (J kg-1 K-1).
         """
-        specific_heat = ((-1.*mixing_ratio + 1.) * consts.U_CP_DRY_AIR +
-                         mixing_ratio * consts.U_CP_WATER_VAPOUR)
-        specific_heat.rename('specific_heat_capacity_of_moist_air')
+        specific_heat = ((-1.*mixing_ratio + 1.) * consts.U_CP_DRY_AIR.points +
```

Comment:
Rather than use the Iris AuxCoord you could use the underlying constant CP_DRY_AIR, CP_WATER_VAPOUR. The Iris AuxCoords are used to keep the associated units with the constants. However in this function we are not making use of the Units.

Reply:
Thank you.